packages/viewtoggle/src/react/__specs__/__snapshots__/storyshots.spec.ts.snap

```diff
@@ -10,11 +10,11 @@ exports[`Storyshots active default first 1`] = `
     data-css-w8tgx1=""
     style={Object {}}
   >
-    <span
+    <div
```

Comment:
This indicates that the render of this component was changed.  But I have a really bad diff, so I can't tell where or why easily.